### DISCUSSION TITLE
Rename reports repo

### DISCUSSION THREAD

**Discussion by ryanmerolle** [netbox-community/netbox#11904] [Polls]
The reports repo contains the following customizations:

- custom_validators
- reports
- scripts
- other (random scripts not used by in NetBox core, but can be used to interact with NetBox)

It seems like the current name "reports" may not accurately reflect the full scope of the project.  What are your suggestions for a name change?

**Comment by XioNoX** [comment]
We use netbox-extras: https://gerrit.wikimedia.org/r/plugins/gitiles/operations/software/netbox-extras/+/refs/heads/master (soon with validators too)

**Comment by ryanmerolle** [comment]
The repo has been renamed to [customizations](https://github.com/netbox-community/customizations)
